.github/workflows/test_iasworld_data.yaml

```diff
@@ -0,0 +1,141 @@
+name: test-iasworld-data
+
+# This workflow is not configured to run on PRs, because PRs have their own CI
+# process defined by the `build-and-test-dbt` workflow. Instead, the Data
+# Department's Spark data extraction process dispatches it via API once per
+# day after finishing the daily ingest of iasWorld data. For more detail, see:
+#
+# https://github.com/ccao-data/service-spark-iasworld
+on:
+  workflow_dispatch:
+    inputs:
+      select:
+        description: >
+          Optional space-separated list of tests to run (defaults to all
+          iasWorld data tests)
+        required: false
+        type: string
+      selector:
+        description: >
+          Optional dbt selector representing tests to run (takes precedence
+          over the above list of tests if both are present)
+        required: false
+        type: string
+      upload_test_results:
+        description: Upload test results to S3
+        required: false
+        default: false
+        type: boolean
+
+env:
+  PYTHONUNBUFFERED: "1"
+  UV_SYSTEM_PYTHON: 1
+
+jobs:
+  test-iasworld-data:
+    runs-on: ubuntu-latest
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint
+    # so that we can authenticate with AWS
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Validate and parse input variables
+        id: parse-inputs
+        run: |
+          # Default to no select option, which will fall back to the default
+          # behavior of the underlying Python script
+          SELECT_OPTION=""
```

Comment:
This block is different from the old implementation of `test-dbt-models`: Now that this workflow is dedicated to running iasWorld tests, we can fall back to the default behavior of the underlying `run_iasworld_data_tests.py` script, which means we can avoid passing in the `--select` or `--selector` options unless the caller specifies one of them.